docs/pages/repo/docs/core-concepts/monorepos/task-dependencies.mdx

```diff
@@ -166,13 +162,28 @@ To accomplish this, we can set up a pipeline like this:
 ![](/images/docs/task-graph-with-placeholder-task.png)
 
 In this pipeline, we declare a "synthetic" task called `topo`. Since we don't
-have a `topo` script in any of our `package.json` files, on `turbo test` the
-pipeline will go straight to running `test` scripts in parallel, meeting our first requirement.
+have a `topo` script in any of our `package.json` files, on `turbo test`, the
```

Comment:
"...on `turbo test`, the pipeline" read awkwardly for me. Maybe "...the `turbo test` pipeline will..."